lib/rubocop/ast_node.rb

```diff
@@ -0,0 +1,17 @@
+# encoding: utf-8
+
+require 'astrolabe/node'
+
+module Astrolabe
+  # RuboCop's extensions to Astrolabe::Node (which extends Parser::AST::Node)
+  #
+  # Contribute as much of this as possible to the `astrolabe` gem
+  # If any of it is accepted, it can be deleted from here
+  #
+  class Node
+    def multiline?
+      expr = loc.expression
+      expr && expr.source.include?("\n")
```

Comment:
I do not think this is accurate at determining if all types of nodes are multiline. Strings being the main issue as they could include `"\n"` within the string while still being on one line.

Something similar came up in a recent PR. If I remember correctly, the code used was
```ruby
node.loc.expression.first_line != node.loc.expression.last_line
```